webpack.config.js

```diff
@@ -6,8 +6,9 @@ const { DefinePlugin } = require( 'webpack' );
 const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
 const TerserPlugin = require( 'terser-webpack-plugin' );
 const postcss = require( 'postcss' );
-const { get, escapeRegExp, compact } = require( 'lodash' );
-const { basename, sep } = require( 'path' );
+const { escapeRegExp, compact } = require( 'lodash' );
```

Comment:
Changes in this file belong to #29537 — please ignore them when reviewing this PR.

Reply:
You can always base your PR on a different branch to avoid exposure to unrelated changes.

Reply:
These commits are related, as I need the changes from that PR to have JS loaded in the frontend.

We could say that the present PR is the reason that #29537 exists.

Reply:
@gziolo did you mean changing the base branch from `trunk` to `block-frontend-js-poc` ? https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request